### PR TITLE
Various improvements to whatthetrain

### PR DESCRIPTION
**Pull request by sylvestre**
Closes issue #6

https://product-details.mozilla.org/1.0/firefox_versions.json

### CODE REVIEW

**Review by luser — CHANGES_REQUESTED** [review]
I think you should change this a little bit more--right now the existing code will make one request per branch, which is silly now that all the data comes from the same URL. You should just have it fetch the data once and then loop over branches in the response.

Otherwise this looks good, thanks!

**Comment by luser** [comment]
The other two changesets look fine. I'm not sure that having the version of ESR.next is super valuable, but once you've removed Aurora it should be fine in terms of information density.

**Comment by sylvestre** [comment]
Yeah, I have been lazy ;) I will fix that!

**Comment by sylvestre** [comment]
@luser sorry for the ping. I would like this to land to update moz-corsica! Thanks!

**Comment by luser** [comment]
FYI we might also need to get https://github.com/luser/whatthetrain/issues/21 fixed to make Corsica happy.

File: trains.js [hunk 1]
```diff
@@ -11,25 +13,31 @@ var RELEASE_CALENDAR = "mozilla.com_2d37383433353432352d3939@resource.calendar.g
...
-function getVersion(responseText, keyJSON) {
-  contentJSON = JSON.parse(responseText);
-  return contentJSON[keyJSON].replace(/[ab]\d+$/, '').replace(/esr$/, '');
+function getVersion(jsonContent, keyJSON) {
+  return jsonContent[keyJSON].replace(/[ab]\d+$/, '').replace(/esr$/, '');
 }
 
-function fetchData(branch, keyJSON, description) {
+function fetchData(jsonContent, branch, keyJSON, description) {
+  if (Object.keys(jsonContent).length > 0 && jsonContent.constructor === Object) {
```
Comment (luser) [inline trains.js]: This isn't quite the way I would have written it, but I'll just do a fixup commit after landing this vs. holding your PR up longer.

**Pull request merged**